plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1063,6 +1064,250 @@ public void testCreatePartitionedTableWithQuotedIdentifierCasing(String columnNa
         }
     }
 
+    @Test
+    public void testSortByAllTypes()
```

Comment:
Could you please add a test with UPDATE, DELETE and MERGE? They should work but just in case